server/connection.go

```diff
@@ -159,6 +237,8 @@ func (s *Server) DeleteConnection(id int) error {
 
 	delete(s.connections, id)
 
+	s.ResetGraph(c)
```

Comment:
can we put a TODO here to mark it as something we think about more in the future?